logzio-sender/src/main/java/io/logz/sender/HttpsRequestConfiguration.java

```diff
@@ -56,7 +56,7 @@ public boolean isCompressRequests() {
     private final boolean compressRequests;
 
 
-    public HttpsRequestConfiguration(String logzioToken,
+    private HttpsRequestConfiguration(String logzioToken,
```

Comment:
If this breaks backward compatibility - you need to mention that in the release notes